core/services/keystore/cosmos_test.go

```diff
@@ -1,30 +1,31 @@
 package keystore_test
 
 import (
+	"context"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/utils"
 	"github.com/smartcontractkit/chainlink/v2/core/internal/cltest"
-	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils/configtest"
+	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils"
 	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils/pgtest"
 	"github.com/smartcontractkit/chainlink/v2/core/services/keystore"
 	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/cosmoskey"
 )
 
 func Test_CosmosKeyStore_E2E(t *testing.T) {
 	db := pgtest.NewSqlxDB(t)
-	cfg := configtest.NewTestGeneralConfig(t)
 
-	keyStore := keystore.ExposedNewMaster(t, db, cfg.Database())
-	require.NoError(t, keyStore.Unlock(cltest.Password))
+	keyStore := keystore.ExposedNewMaster(t, db)
+	require.NoError(t, keyStore.Unlock(testutils.Context(t), cltest.Password))
 	ks := keyStore.Cosmos()
 	reset := func() {
+		ctx := context.Background() // Executed during cleanup
```

Comment:
Would it benefit to be `testutils.Context(t)`? I'm not 100% on my understanding of how go test cleanup operates, but i've had some friction with debugging tests that hang/fail in cleanup vs in the test execution.

Reply:
No it can't be a normal ctx since we execute it during t.Cleanup, when the context may already be done.